docs/Language Feature Status.md

```diff
@@ -0,0 +1,31 @@
+# Language Feature Status
+
+## C# 7.0 and VB 15
+
+| Feature | Branch | State | Owners |
+| ------- | ------ | ----- | ------ | ----- |
+| Async Main | none  | Feature Specification | [tyoverby](https://github.com/tyoverby), [agocke](https://github.com/agocke) |
+| Address of Static | none | Feature Specification | |
+| Binary Literals | [future](https://github.com/dotnet/roslyn/tree/future)  | Finishing |  |
+| Digit Separators | [future](https://github.com/dotnet/roslyn/tree/future)  | Finishing | |
+| [Local Functions](https://github.com/dotnet/roslyn/blob/future/docs/features/local-functions.md) | [future](https://github.com/dotnet/roslyn/tree/future)  | Finishing | [agocke](https://github.com/agocke), [jaredpar](https://github.com/jaredpar), [vsadov](https://github.com/vsadov) |
+| [Pattern Matching](https://github.com/dotnet/roslyn/blob/future/docs/features/patterns.md) | [features/patterns](https://github.com/dotnet/roslyn/tree/features/patterns) | Prototyping | [gafter](https://github.com/gafter), [alekseyts](https://github.com/alekseyts), [agocke](https://github.com/agocke) |
+| Records | none | Feature Specification | [jcouv](https://github.com/jcouv) |
+| Ref Returns | [future](https://github.com/dotnet/roslyn/tree/future) | Finishing | [vsadov](https://github.com/vsadov), [agocke](https://github.com/agocke), [jaredpar](https://github.com/jaredpar) |
+| Source Generation | [future](https://github.com/dotnet/roslyn/tree/features/generators) | Prototyping | [cston](https://github.com/cston), [vsadov](https://github.com/vsadov) |
+| Throw Expr | [features/patterns](https://github.com/dotnet/roslyn/tree/features/patterns) | Prototyping | [agocke](https://github.com/agocke), [tyoverby](https://github.com/tyoverby), [gafter](https://github.com/gafter) |
+| Tuples | [features/tuples](https://github.com/dotnet/roslyn/tree/features/tuples) | Prototyping | [vsadov](https://github.com/vsadov), [jcouv](https://github.com/jcouv) |
+| Out var | none | Feature Specification | [alekseyts](https://github.com/alekseyts) |
+| With Exprs | none | Feature Specification | [gafter](https://github.com/gafter) |
+
+## C# 7.0 and VB 15 + 1
+
+| Feature | Branch | State | Owners |
+| ------- | ------ | ----- | ------ | ----- |
+| Protected Private | [features/privateprotected](https://github.com/dotnet/roslyn/tree/features/privateprotected) | Prototyping | |
+| [Non-null Ref Types](https://github.com/dotnet/roslyn/blob/features/NullableReferenceTypes/docs/features/NullableReferenceTypes/Nullable%20reference%20types.md) | [features/NullableReferenceTypes](https://github.com/dotnet/roslyn/tree/features/NullableReferenceTypes) | Prototyping | [alekseyts](https://github.com/alekseyts) |
+
+# FAQ
+
+- **Is target version a guarantee?**: No.  It's explicitly not a guarantee.  This is just the plan of record to the best of our knowledge at this time.
```

Comment:
This conflicts with your statement that this is what is currently being worked on. It cannot be both a snapshot of our activity and a plan of record. If it is the latter, it will require LDM involvement.